ood-portal-generator/lib/ood_portal_generator.rb

```diff
@@ -26,5 +29,25 @@ def scl_apache?
       return false if ("#{env['ID']} #{env['ID_LIKE']}" =~ /rhel/ && env['VERSION_ID'] =~ /^8/)
       true
     end
+
+    def fqdn
+      Socket.gethostbyname(Socket.gethostname).first
```

Comment:
Is there a situation you can imagine that this would return a value that is not desired?

Reply:
I've never experienced that but I think it might only do that if host lookups are broken but I don't know if this makes a call to DNS or if it does something like glibc lookups using nsswitch which does not make external calls usually for local host names.